glide-core/src/client/mod.rs

```diff
@@ -415,6 +420,28 @@ impl Client {
         .boxed()
     }
 
+    pub fn send_pipeline<'a>(
```

Comment:
please add description so it would be clear how this function is different from send_transaction